PubNub/Data/Managers/PNPublishSequence.m

```diff
@@ -369,30 +376,6 @@ - (void)subscribeOnNotifications {
 #endif // TARGET_OS_OSX
 }
 
-- (void)unsubscribeFromNotifications {
```

Comment:
I was actually referring to no longer having the `UIApplicationDidResignActive`/`DidEnterBackground` notifications. Why are those going away?